src/Raven.Server/Program.cs

```diff
@@ -350,7 +350,7 @@ private static void VerifyLicenseVersion(LicenseStatus licenseStatus, License li
                 if (licenseStatus.Version.Major >= 6)
                 {
                     serverStore.LicenseManager.OnBeforeInitialize += () =>
```

Comment:
There is another place where we get the license from the API.

Reply:
done